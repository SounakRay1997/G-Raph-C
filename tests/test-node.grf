node a;
node b;
int i;

int main(){
    int c;
    c=0;
    i=0;
    b = makeNode("node b", 0);
    while(c<7){
        setNodeValue(b,c);
        c=c+1;

        }
    while(i<4){
         
         i=i+1;
    }
    
    a = makeNode("asdf",10);
    print(getNodeValue(a));
    print(getNodeValue(b));
    print(getNodeLabel(a));
    print(getNodeLabel(b));
    return 0;
}