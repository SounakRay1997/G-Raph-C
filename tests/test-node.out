10
6
asdf
node b
